test/metabase/api/user_test.clj

```diff
@@ -147,11 +175,59 @@
+                  group-ids->sets
+                  mt/boolean-ids-and-timestamps
+                  (map #(dissoc % :is_qbnewb :last_login)))))
+      (is (= (->> [{:email                  "trashbird@metabase.com"
+                    :first_name             "Trash"
+                    :last_name              "Bird"
+                    :is_active              false
+                    :group_ids              #{(u/the-id (group/all-users))}
+                    :personal_collection_id true
+                    :common_name            "Trash Bird"}
+                   {:email                  "crowberto@metabase.com"
+                    :first_name             "Crowberto"
+                    :last_name              "Corv"
+                    :is_superuser           true
+                    :group_ids              #{(u/the-id (group/all-users))
+                                              (u/the-id (group/admin))}
+                    :personal_collection_id true
+                    :common_name            "Crowberto Corv"}
+                   {:email                  "lucky@metabase.com"
+                    :first_name             "Lucky"
+                    :last_name              "Pigeon"
+                    :group_ids              #{(u/the-id (group/all-users))}
+                    :personal_collection_id true
+                    :common_name            "Lucky Pigeon"}
+                   {:email                  "rasta@metabase.com"
+                    :first_name             "Rasta"
+                    :last_name              "Toucan"
+                    :group_ids              #{(u/the-id (group/all-users))}
+                    :personal_collection_id true
+                    :common_name            "Rasta Toucan"}]
+                  (map (partial merge user-defaults))
+                  (map #(dissoc % :is_qbnewb :last_login)))
+             (->> (mt/user-http-request :crowberto :get 200 "user", :status "all")
                   (filter mt/test-user?)
                   group-ids->sets
                   mt/boolean-ids-and-timestamps
                   (map #(dissoc % :is_qbnewb :last_login))))))))
 
+(deftest user-list-limit-test
+  (testing "GET /api/user?limit=1&offset=1"
+    (testing "Limit and offset pagination have to have both limit and offset"
+      (is (= "When including a limit, an offset must also be included." (mt/user-http-request :crowberto :get 400 "user" :limit "1")))
+      (is (= "When including an offset, a limit must also be included." (mt/user-http-request :crowberto :get 400 "user" :offset "1"))))
+    (testing "Limit and offset pagination works for user list"
+      (is (= [{:id          (mt/user->id :lucky)
+               :email       "lucky@metabase.com"
+               :first_name  "Lucky"
+               :last_name   "Pigeon"
+               :common_name "Lucky Pigeon"}]
+             (->> (mt/user-http-request :rasta :get 200 "user" :limit "1" :offset "1")
```

Comment:
Does it make sense to have this test use a limit > 1 to make sure there isn't some crazy issue with the list having count > 1?